libs/sdk-backend-tiger/src/convertors/fromBackend/AutomationConverter.ts

```diff
@@ -159,8 +159,8 @@ const convertAlert = (
                 operator: comparison.operator as IAlertComparisonOperator,
                 left: {
                     id: comparison.left.localIdentifier,
-                    title: comparison.left.title,
-                    format: comparison.left.format,
+                    title: comparison.left.title ?? undefined,
```

Comment:
Why is this needed?

Reply:
Api now accept null values here but we want only undefined ....